x-pack/plugins/infra/public/pages/metrics/metrics_explorer/hooks/use_metrics_explorer_options.ts

```diff
@@ -122,6 +123,20 @@ export const useMetricsExplorerOptions = () => {
     DEFAULT_CHART_OPTIONS
   );
   const [isAutoReloading, setAutoReloading] = useState<boolean>(false);
+
+  const { metricThresholdPrefill } = useAlertPrefillContext();
+
+  useEffect(() => {
+    if (metricThresholdPrefill) {
+      const { setMetrics, setGroupBy, setFilterQuery } = metricThresholdPrefill;
+      const { metrics, groupBy, filterQuery } = options;
+
+      setGroupBy(groupBy);
+      setFilterQuery(filterQuery);
+      setMetrics(metrics);
+    }
+  }, [options, metricThresholdPrefill]);
+
```

Comment:
Can we add a test to `x-pack/plugins/infra/public/pages/metrics/metrics_explorer/hooks/use_metrics_explorer_options.test.tsx` that covers this?